### ISSUE TITLE
Wanted to give time wise restriction to the files shared

### ISSUE THREAD

**Issue by precihole** [frappe/drive#131]
So we have a scenario in which we share our designs with a client, and we wanted to limit their access to files for a set length of time, as well as the number of times they could open the same file.

**Comment by Zahidshakil** [comment]
Any development on it

**Comment by midhat2020** [comment]
@safwansamsudeen Hey, so are you working on folder actions?

**Comment by safwansamsudeen** [comment]
@midhat2020 no, not right now - though I'd be open to a PR if it's of sufficiently high quality.
